config/routes.rb

```diff
@@ -89,6 +89,8 @@
   post '/profile/reactivate' => 'users/reactivate_profile#create'
   get '/profile/verify' => 'users/verify_profile#index', as: :verify_profile
   post '/profile/verify' => 'users/verify_profile#create'
+  get '/profile/verify_phone' => 'users/verify_profile_phone#index', as: :verify_profile_phone
```

Comment:
@monfresh for consistency, we've started renaming all the `/profile` URLs to `/account` and setting up redirects for old ones (see #1407). Since this is brand-new can we rename it to `/account/verify_phone` before it lands? (Also appears there are merge conflicts related to the rename....oops)